resources/user.py

```diff
@@ -224,3 +227,23 @@ def get(self):
             } for user in users]
 
         return {"users": ret}, 200
+
+class UserInvite(Resource):
+    @jwt_required
+    def post(self):
+        data = request.json
+        temp_password = ''.join(random.choice(string.ascii_lowercase) for i in range(8))
+        user = UserModel(
```

Comment:
I would like to see the create method used here; otherwise this endpoint can be prone to bugs. However, this will require a UserSchema which does not exist yet. We could either do this now or create an issue for it.

Reply:
FYI: A basic UserSchema exists now

Reply:
I used the create method now. I had trouble with the user schema at first, but I think I got it working! I was trying to use the User Register Schema which doesn't allow a role.

Reply:
Yes the UserRegister schema is special. I'm glad you got it figured out.